### PR TITLE
Add a usage guide that actually describes how to set up the dashboard

### PR DESCRIPTION
**Pull request by jerzywilczek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,128 @@ For available metrics please refer to the latest core's master branch
...
+  opts = [strategy: :one_for_one, name: MembraneLive.Supervisor]
+  Supervisor.start_link(children, opts)
+end
+```
+
+After this step, your application should automatically log all configured events to the database. If you decided to use `docker-compose`, you don't need to do anything else. If you decided to set up everything by yourself, there's one more step left.
+
+### Running `membrane_dashboard`
+
+Fortunately, this is rather simple. You need to get all the dependencies:
```
Comment (varsill) [inline README.md:134]: How about emphasizing that this one needs to be done in the directory where you have cloned the `membrane_dashboard`? I know it should be quite obvious, but the previous steps were done in the application directory :D

File: README.md [hunk 2]
```diff
@@ -29,6 +29,128 @@ For available metrics please refer to the latest core's master branch
...
+-e POSTGRES_USER=postgres \
+-e POSTGRES_PASSWORD=postgres \
+--name dashboard-timescale \
+-d -p 5432:5432 \
+timescale/timescaledb:latest-pg14
+```
+
+### Wiring a membrane pipeline to a `membrane_timescaledb_reporter`
+
+This is the most complicated part of all of this. First, add `membrane_timescaledb_reporter` to your `mix.exs`:
```
Comment (varsill) [inline README.md:64]: I would add an explanation that these changes need to be done in your application directory, where the membrane pipeline is run

**Review by Qizot — APPROVED** [review]
LGTM

**Review by varsill — APPROVED** [review]
LGTM 🥇

**Pull request merged**